js/src/settings.js

```diff
@@ -48,9 +45,8 @@ jQuery(
 		var wizardSelectmenuWidth = '100%';
 
 		// Inject flag image when jQuery UI selectmenu is created or an item is selected.
-		// jQuery UI 1.12 introduce a wrapper inside de li tag which is necessary to selectmenu widget to work correctly.
+		// jQuery UI 1.12 introduces a wrapper inside de li tag which is necessary to selectmenu widget to work correctly.
```

Comment:
The comment could be deleted since there is no backward compatibility, am I right?